src/Proj4.jl

```diff
@@ -101,11 +99,14 @@ end
 """
 Return true if the projection is a geocentric coordinate system
 """
-function is_gencent(proj::Projection)
+function is_geocent(proj::Projection)
     @assert proj.rep != C_NULL
     ccall((:pj_is_geocent, libproj), Cint, (Ptr{Void},), proj.rep) != 0
 end
 
+"Get projection definition string in the proj.4 plus format"
+proj4string(proj::Projection) = _get_def(proj)
```

Comment:
Did you consider the version I pushed to master before I raised this yesterday?  In that, I just overloaded `Base.print(::IO,::Projection)`, which seems like it might be the right thing (though I'm not completely certain).  I had to dig surprisingly deeply into the documentation to work out precisely which functions seem like the right ones to use.  Here's the reference for the distinction between `show()` and `print()`.

http://julia.readthedocs.org/en/latest/stdlib/io-network/#Base.print
http://julia.readthedocs.org/en/latest/stdlib/io-network/#Base.show